financial_transcripts/rag-cosmos-mongo/flow.dag.yaml

```diff
@@ -151,6 +150,18 @@ nodes:
     path: CollectionFilter.py
   inputs:
     input1: ${QueryParser.output}
+- name: QueryRewriter
+  type: llm
+  source:
+    type: code
+    path: QueryRewriter.jinja2
+  inputs:
+    deployment_name: gpt-35-turbo
+    max_tokens: 5000
+    query: ${inputs.query}
+    chat_history: ${inputs.chat_history}
+  connection: aoai_connection
+  api: chat
 node_variants:
   DetermineReply:
     default_variant_id: variant_0
```

Comment:
@alsavelv @HosseinHeris I noticed there is a variant for the DetermineReply node which is inconsistent with `rag-azure-search` and `rag-cosmos-postgres`. Please remove the variant or be add to the other samples

Reply:
That variant is there by design. I wanted to show case an example for tracking prompt variation.

Reply:
@journeyman-msft : are you able to get an answer with the current main?

Reply:
@HosseinHeris Then shouldn't all 3 rag-azure-seasrch, rag-cosmos-mongodb, rag-postgres have a variant? Yes they all ran from promptflow, but follow up success depend on the question

Reply:
Variant is only one of the LLMs calls, just to show case how to do it. We do not need to replicate everything in all the samples unless it is a significant improvement, like query rewriter. 
By the way @alsavelv: now that the default is gpt4, please make sure the variant is gpt3.5 turbo.

Reply:
@HosseinHeris  are you suggesting that I add one more variant to QueryRewriter? to clarify, I only put gpt4 as model for that node. DetermineReply still uses gpt3.5

Reply:
per discussion with Hossein, changing the model to gpt-4-32k for all nodes and making gpt3.5 non-default variant for DetermineReply on the flow where we have variants (cosmos-mongodb)

Reply:
the switch to gpt-4-32k occasionally still leads to failures, but we agreed to address it in a different PR